functional-tests/src/test/groovy/spec/Tailings.groovy

```diff
@@ -38,5 +50,46 @@ class Tailings extends GebReportingSpec {
         then: "the reports are visible"
         assert tailingsTab.document0Name != null
     }
+
+    //ToDo: THIS TEST ONLY WORKS IN CHROME AND CHROME HEADLESS.  WORK IN GebConfig to fix other browsers
+    def "Scenario: User is able to upload a TSF Report"(){
+        when: "User navigates to the TSF tab and clicks the upload icon"
+        tailingsTab.tabSelect.click()
+
+        and: "User opens modal and uploads a valid file type"
+        def uploadedFile = dir.newFile(Const.TEST_FILE_NAME) << Const.TEST_FILE_CONTENT
+        tailingsTab.addTailingsDocButtons[0].click()
+        tailingsTab.uploadField = uploadedFile.absolutePath
+
+        then: "The doc upload complete message is shown"
+        assert tailingsTab.uploadCompleteMessage != null
+
+    }
+
+    //ToDo: THIS TEST ONLY WORKS IN CHROME AND CHROME HEADLESS.  WORK IN GebConfig to fix other browsers
```

Comment:
TODO label on comments should be all-caps

Reply:
Changed